aten/src/ATen/NestedTensorImpl.h

```diff
@@ -83,8 +89,10 @@ struct TORCH_API NestedTensorImpl : public c10::TensorImpl {
   // Must be called after any changes to our dim() to sync the state
   // to TensorImpl.
   void refresh_dim();
-
-  at::Tensor buffer_;
+  // Store the size of the buffer for use in get_buffer().
+  // get_buffer constructs a flat, contiguous tensor from the NestedTensor
+  // storage
+  std::vector<int64_t> buffer_size_;
```

Comment:
Not a dumb question this will be a vector of one element which is the size. And I missed this overload:
```
/// Construct an ArrayRef from a single element.
template <typename T>
ArrayRef<T> makeArrayRef(const T& OneElt) {
  return OneElt;
}
```
This is where we use the buffer_size in get_buffer(). I will update now